Planner/qSlicerPlannerModuleWidget.cxx

```diff
@@ -1201,12 +1202,12 @@ void qSlicerPlannerModuleWidgetPrivate::cancelCut(vtkMRMLScene* scene)
 //Delete model and associated nodes from scene
 void qSlicerPlannerModuleWidgetPrivate::deleteModel(vtkMRMLModelNode* node, vtkMRMLScene* scene)
 {
-  vtkMRMLSubjectHierarchyNode* hnode = vtkMRMLSubjectHierarchyNode::SafeDownCast(
+  vtkMRMLSubjectHierarchyNode* shNode = vtkMRMLSubjectHierarchyNode::SafeDownCast(
                                        vtkMRMLHierarchyNode::GetAssociatedHierarchyNode(node->GetScene(), node->GetID()));
 
-  if(hnode)
+  if(shNode)
   {
-    scene->RemoveNode(hnode);
+    scene->RemoveNode(shNode);
```

Comment:
@sjh26 @bpaniagua 

1. Instead of using `vtkMRMLHierarchyNode::GetAssociatedHierarchyNode` to get a SH node, I should use the scene to get the SH node, and then use the SH node to get the parent item of the node.
2. Instead of removing a subject hierarchy node from the scene, I should use `vtkMRMLSubjectHierarchyNode::RemoveItem` to remove the parent item and it's descendants.

Reply:
I have implemented these changes in https://github.com/KitwareMedical/OsteotomyPlanner/pull/118/commits/7b61b08bcf522ddc43c569a8bde5465f6d376aca.